src/test/java/io/aiven/kafka/connect/gcs/GcsSinkTaskTest.java

```diff
@@ -109,6 +110,35 @@ final void basic() {
             readSplittedAndDecodedLinesFromBlob("topic1-1-40", false, 0));
     }
 
+    @Test
+    final void basicNewTemplate() {
```

Comment:
Maybe sth. like `withTimebasedGrouping` would be a better name here

Reply:
👍